lesson9/lesson9.js

```diff
@@ -0,0 +1,36 @@
+const ul = document.getElementById('js-list');
+const listItems = [
+  { to: "bookmark.html", img: "bookmark-icon.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "mail-icon.png", alt: "画像2", text: "メッセージ" }
+];
+const loadingImg = document.createElement('img');
+loadingImg.src = "loading-circle.gif";
+loadingImg.id = "loading-img";
+ul.appendChild(loadingImg);
```

Comment:
IMO
I think it's better to make these into functions, as you did at first.

Reply:
I'm curious, may I know why you think it's better?

Reply:
I thought that would be better for showing and hiding the loading icon again.
Am I perhaps misunderstanding something? If so, I am sorry.

Reply:
Please don't be sorry! I was just curious about your thinking process. I just thought appending/removing the image from DOM directly was more simple than creating functions.